### PR TITLE
ladspaPlugins: 0.4.15 -> 0.4.17

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] OS X
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by magnetophon** [comment]
Does anyone know why travis failed?

File: pkgs/applications/audio/ladspa-plugins/default.nix [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
   };
-  
-  buildInputs = [fftw ladspaH pkgconfig];
 
-  postInstall =
-    ''
-      mkdir -p $out/share/ladspa/
-      ln -sv $out/lib/ladspa $out/share/ladspa/lib
-    '';
+  buildInputs = [ autoreconfHook fftw ladspaH libxml2 pkgconfig perl perlPackages.XMLParser ];
```
Comment (FRidh) [inline pkgs/applications/audio/ladspa-plugins/default.nix]: you can remove perl and use perlPackages.perl instead. That way we can guarantee the same perl is used.

**Comment by magnetophon** [comment]
@FRidh OK.

**Comment by FRidh** [comment]
> Does anyone know why travis failed?

https://github.com/NixOS/nixpkgs/commit/ab0ac18279d6497bdeb616cbe7cd0797d6534739

I hope my fix was correct.

**Comment by magnetophon** [comment]
@FRidh Thanks for your help!

Weird: github reports the file as a new ``.nix`` file (but no mention of a rename), travis doesn't find it, and when I checkout that commit locally, the file shows up as ``pkgs/misc/jackaudio/unstable.git``.

**Comment by magnetophon** [comment]
https://github.com/NixOS/nixpkgs/pull/20089

**Comment by FRidh** [comment]
@magnetophon correct, I used git mv but apparently forgot to stage the old file.

**Comment by magnetophon** [comment]
@FRidh apparently [travis didn't get the message](https://travis-ci.org/NixOS/nixpkgs/jobs/172600933#L170).

**Pull request merged**